README.md

```diff
@@ -17,96 +12,42 @@ Clone the source code:
-
-For ease of testing, it is recommended to download [Insomnia API Client and Design Tool](https://insomnia.rest) for sending HTTP requests to the server and [Robo 3T](https://robomongo.org) as a GUI for viewing the Mongo database.
-
-When sending requests, ensure that the `"Content-type": "application/json+fhir"` header is set.
-
-### CRUD Operations
+Once you have the necessary dependencies installed in the `frontend` and `backend` directories, you can run the following in the root directory:
 
-This server currently supports the following CRUD operations:
+To start the the frontend application and backend server in parallel:
 
-- Read by ID to endpoint: `4_0_1/<resourceType>/<resourceId>`
-  _More functionality coming soon!_
-
-### Search
-
-The Measure Repository Service server supports `Library` and `Measure` resource search by the parameters specified in the shareable measure repository section of the [HL7 Measure Repository Docs](https://build.fhir.org/ig/HL7/cqf-measures/measure-repository-service.html#shareable-measure-repository). These parameters are:
-
-- description
-- identifier
-- name
-- status
-- title
-- url
-- version (can appear only in combination with a url search)
-
-### Package
-
-The Measure Repository Service server supports the `Library` and `Measure` `$package` operation with the `id` and `identifier` parameters and SHALL parameters specified in the shareable measure repository section of the [HL7 Measure Repository Docs](https://build.fhir.org/ig/HL7/cqf-measures/measure-repository-service.html#publishable-measure-repository). Accepted parameters are:
-
-- id
-- url
-- identifier
-- version
-
-### Data Requirements
-
-The Measure Repository Service server supports the `Measure` and `Library` `$data-requirements` operations with the SHALL parameters specified in the publishable measure repository section of the [HL7 Measure Repository Docs](https://build.fhir.org/ig/HL7/cqf-measures/measure-repository-service.html#publishable-measure-repository). SHALL parameters are:
-
-- id
-- url
-- version
-- identifier
+```bash
+npm run start:all
+```
 
-Supported optional parameters for `Measure` are:
+This starts up the Measure Repository service at `localhost:3000` and you can open the Measure Repository Service frontend application by navigating to http://localhost:3001 in your browser.
 
-- periodStart
-- periodEnd
+To start only the frontend:
 
-### Submit
-The Measure Repository Service server supports the `$submit` operation for `Measure` and `Library` resources, as specified in the Authoring Measure Repository section of the [HL7 Measure Repository Docs](https://build.fhir.org/ig/HL7/cqf-measures/measure-repository-service.html#authoring-measure-repository). The operation does not take in any parameters. 
+```bash
+npm run start:frontend
+```
 
-## License
+To start only the backend:
 
-Copyright 2022 The MITRE Corporation
```

Comment:
Should we still have the Apache License at the root dir as well or nah? I don't think we have a license specified in the frontend README - I assume the license at the root would apply to everything, right?